.github/workflows/scan-vulns.yaml

```diff
@@ -0,0 +1,68 @@
+name: scan_vulns
+on:
+  push:
+    paths-ignore:
+      - "docs/**"
+      - "library/**"
+      - "**.md"
+  pull_request:
+    paths-ignore:
```

Comment:
Does this mean this will run for every PR? Regardless of release branch or not?

Reply:
correct, would that be desired? the scan takes about 3min, if there are any PR that introduces any vulnerbility, we ll be able to catch it early